README.md

```diff
@@ -1,2 +1,14 @@
 # New Chip Test Board Gui using Python
 Prototype for a new python based GUI for the Chip Test Board
+
+## Format code
+
+```
+yapf --style pyproject.toml . --recursive  -m -i
```

Comment:
as mentioned above this could be run using "make format" then the user doesn't need to remember the exact syntax and if we change formatter the command can remain the same